src/components/HealthCardDisplay/iisstandards.js

```diff
@@ -5,6 +5,15 @@ import cvxXml from './iisstandards_cvx.xml';
 let tradenames = null;
 let cvx = null;
 
+function decodeEntity(xmlString) {
```

Comment:
This looks good. Would also create an encodeEntity that is the same in reverse, to keep it clean.

Reply:
Also would add tests to confirm this functionality, see iisstandards.test.js in main branch for jest examples.

Reply:
The `decodeEntity` function no longer exists, per discussion below. I did move the `.replace`s used for the initial character escaping into a separate function so that they could be tested in isolation against mock XML data.

Reply:
Updated with a test to check the reverse, that we're not encoded already encoded characters. This failed, which required a fix to the regex.